rundnsmasq.sh

```diff
@@ -15,13 +14,9 @@ mkdir -p /shared/html/pxelinux.cfg
 # Copy files to shared mount
 cp /tftpboot/undionly.kpxe /tftpboot/ipxe.efi /tftpboot/snponly.efi /shared/tftpboot
 
-# Copy IPv4 or IPv6 config
-cp /etc/dnsmasq.conf.ipv$IPV /etc/dnsmasq.conf
+# Template and write dnsmasq.conf
+python -c 'import os; import sys; import jinja2; sys.stdout.write(jinja2.Template(sys.stdin.read()).render(env=os.environ))' </etc/dnsmasq.conf.j2 >/etc/dnsmasq.conf
```

Comment:
nit: it would be a bit shorter to just hardcode file names in python and avoid using sys.std*.